deployment/live/gcp/static-ct-staging/cloudbuild/terragrunt.hcl

```diff
@@ -0,0 +1,26 @@
+terraform {
+  source = "${get_repo_root()}/deployment/modules/gcp//cloudbuild/preloaded"
+}
+
+locals {
+  env          = path_relative_to_include()
```

Comment:
Is it likely to have another `env` under the `static-ct-staging` directory? Consider merging this file with deployment/live/gcp/static-ct-staging/cloudbuild/staging/terragrunt.hcl?

Reply:
Yes agreed that can be changed - I paused here, but I'm actively thinking about changing this, and maybe having a cloudbuild for the log, and a different one to run the preloader.